src/components/ui/modals/Modals.vue

```diff
@@ -2,102 +2,132 @@
   <div class="modals">
```

Comment:
Update docs a bit. Some things are off.
https://github.com/epicmaxco/vuestic-admin/wiki/Modals

Reply:
@asvae check the doc